docs/diagrams/FitTrackCore.puml

```diff
@@ -0,0 +1,60 @@
+@startuml
+
+title Core Structure of FitTrack\n
+
+participant ": FitTrack" as main
+participant "ui: Ui" as ui
+participant ": CommandParser" as parser
+participant "command: XXXCommand" as cmd
+
+
+main -> main ++: loopCommandExecution()
+
+group do-while [!ExitCommand.isExit(command)]
```

Comment:
Is this representation of loop also acceptable? In our textbook the usual representation is "loop"